sys/log/full/src/log_console.c

```diff
@@ -24,9 +24,28 @@
 #include <cbmem/cbmem.h>
 #include <console/console.h>
 #include "log/log.h"
+#if MYNEWT_VAL(LOG_VERSION) > 2
+#include "tinycbor/cbor.h"
+#include "tinycbor/cbor_buf_reader.h"
+#endif
 
 static struct log log_console;
 
+static int
+console_log_dump_cbor_entry(struct log *log, const void *dptr, uint16_t len)
```

Comment:
You could remove the first parameter to this function (`struct log *log`).

Reply:
done